requirements.in

```diff
@@ -17,3 +17,5 @@ flower
 polyglot
 pyicu
 pycld2
+opencensus
```

Comment:
As [yurishkuro suggests](https://github.com/openzipkin/zipkin/issues/1265#issuecomment-243950361), we can use a straight OpenTracing library, and the basic [opentracing](https://pypi.org/project/opentracing/) library seems easy enough to use. My complaint with opencensus is that it pulls in a bunch of google libraries.

Reply:
We can do that, but that should be in another PR

Reply:
I think it's easier to make the change now, rather than do it down the line, and have to worry about how the libraries and the data they send may be different.

Reply:
This is a request for change after the implementation, which is not OK. The tasks specifies: "We could use OpenCensus for tracing: https://opencensus.io/quickstart/python/tracing/#run-it-locally. Aleph has just integrated this, for similar purposes, I believe - alephdata/aleph#582."
Also, opentracing doesn't support metrics, like opencensus.

Reply:
Ok, I suppose you're right, let's go with opencensus.